lib/elastic_apm/metrics/vm.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  module Metrics
+    # @api private
+    class VM
+      include Logging
+
+      def initialize(config)
```

Comment:
The `config` argument is not captured. Is that intentional? `config` is an `attr_reader` so I'm assuming it should be captured.

Reply:
IIRC, the `Logging` module expects a `config` method to be available. It's something that I eventually want to rewrite.

Reply:
... but it should of course be saved to an ivar, yes, good catch!

Reply:
https://github.com/elastic/apm-agent-ruby/pull/519